### PR TITLE
Fixing lost data bug

### PR DESCRIPTION
**Pull request by da1nerd**
#### Describe what your pull request addresses (Please include relevant issue numbers):

Related to #5805  
- This corrects a bug that was introduced while moving tool initialization into the project selection action. Simple put, I forgot to also move the tool de-initialization into closing a project. Check out the issue for a more detailed explanation.
- added username to the auto commit message
- added active reference to the auto commit message for easier debugging
- added app version to the log files

> NOTE: when this code is eventually merged with downstream (e.g. 1.1.3) care should be taken because `clearLastProject` was renamed (in downstream) to `closeProject`. We don't want to accidentally lose the newly moved tool disconnection logic.

#### Please include detailed Test instructions for your pull request:
- Open any project in wA.
- Align a verse
- leave the wA tool, but do not close tc or change the project.
- open the wA tool again
- Align a different verse (previously this would not be saved)
- Close and Reopen tC
- Finally open the project in wA again and check that the alignments (particularly the last verse you aligned) remains intact.

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/5825)

### CODE REVIEW

**Review by mannycolon — APPROVED** [review]
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/5825)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @neutrinog)

**Review by mannycolon — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/5825)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @neutrinog)

File: src/js/actions/MyProjects/ProjectLoadingActions.js [hunk 1]
```diff
@@ -217,7 +217,18 @@ export function displayTools() {
  * prevent a new project from loading
  */
 export function clearLastProject() {
-  return (dispatch) => {
+  return (dispatch, getState) => {
```
Comment (mannycolon) [inline src/js/actions/MyProjects/ProjectLoadingActions.js:220]: @neutrinog So the fix was moving the code to disconnect the tools from componentWillUnmount to here?

Comment (mannycolon) [reply src/js/actions/MyProjects/ProjectLoadingActions.js:220]: lol nvm just read the description

**Review by PhotoNomad0 — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/5825)* status: all files reviewed, 1 unresolved discussion (waiting on @neutrinog)

**Comment by PhotoNomad0** [comment]
@cckozie - verified in https://tc-builds.door43.org/build/release-v1.1.2/translationCore-macos-x64-1.1.2-dfa9985.dmg

This is a fix for the problem that when you exit the WA tool then alignments stop saving to disk until new project selected or app restarted.

**Pull request merged**